server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/diskusage/IndexDiskUsageAnalyzerIT.java

```diff
@@ -63,6 +63,17 @@ public void testSimple() throws Exception {
                 .setSource(doc)
                 .get();
         }
+        final boolean hasNorms = randomBoolean();
```

Comment:
maybe call this "forceNorms"? Because we would still have norms when this variable here is false.

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/76894/commits/1b1599963ef8ea0825405fc14f41058ec1282d1f.